client/src/components/Rooms/MeetingRooms/MeetingRooms.js

```diff
@@ -28,6 +30,8 @@ const MeetingRooms = () => {
 	}, []);
 
 	return (
+		<>
```

Comment:
Might be helpful to give a classname in the future incase we need to identify it in the DOM(might be easier to spot) or if we need give styling for that section. Nothing major tho! 👍